docs/workflows.md

```diff
@@ -317,59 +317,67 @@ We see that our output graph looks signficantly different than the input. That's
       'name': 'DPG Tutorial 2025 Project Workflow'
       'inputs':
       - 'name': 'Workflow Parameters'
-        'section': '/entries/tc_dLqPqR9lqLT1JbFr0IfZ1n39e#/data'
+        'section': '/entries/tc_dLqPqR9lqLT1JbFr0IfZ1n39e/archive#/data'
```

Comment:
I'm confused, didn't you say that this option currently isn't resolved?

Reply:
yes, but i don´t see alternatives. including upload id manually is not the case.

This is still the file coming from the worflow-utilities with smaller changes. I think it should be there, until the utilities are able to produce a better file

Reply:
I see, then this approach is really relying on the utilities working...
It's fine, you can go ahead and merge this, and I'll revisit this paragraph in case it's necessary.

Reply:
in a way yes, but we can also ask attendees to modify the file created even if we don't have any patch yet. Let's see

ok great, i'll merge